apps/parse/lib/parse/calendar_attributes.ex

```diff
@@ -0,0 +1,30 @@
+defmodule Parse.CalendarAttributes do
+  @moduledoc """
+  Parser for GTFS calendar_attributes.txt
+  """
+  @behaviour Parse
```

Comment:
You can `use Parse.Simple` instead, and then you only need to implement `parse_row/1`.